documentation/src/main/asciidoc/topics/ref_endpoints_connectors.adoc

```diff
@@ -18,6 +18,8 @@ and whether clients must present valid TLS certificates.
 * `<memcached-connector socket-binding="memcached"/>` declares a Memcached
 connector that uses the memcached socket binding.
 
+Declaring an empty `<endpoints/>` element implicitly enables the Hot Rod and REST connectors.
```

Comment:
To disable the Hotrod endpoint do I need todo the following?

```
<endpoints>
  <rest-connector />
</endpoints>
```

Reply:
yes